src/items/menu.js

```diff
@@ -132,7 +132,7 @@ export default class MenuPanelItem extends PanelItem {
 
       this.core.make('osjs/contextmenu').show({
         menu: makeTree(this.core, icon, __, [].concat(packages)),
-        position: ev.target.closest('.osjs-panel-item--clickable'),
+        position: this.$element,
```

Comment:
Or just `ev.target`. But either could hit either the image *or* the parent element of that. So a `closest()` type behavior will make it more consistent.

Reply:
Silly me, `currentTarget` is bound to the element where the actual event was registered. So this would probably place it in the correct coordinate as the ContextMenu will calculate the bottom-left position :)

Reply:
Yes, I completely forgot about this myself when I did the original PR. So at least for the sole purpose of consistent menu positioning, this PR is redundant. It does however provide a more robust usage of Hyperapp and may prevent bugs such as the one I had in @osjs/client. So it's totally up to you whether you still want it.

Reply:
The panel is always re-initialized when it is modified, so the Hyperapp usage would not really affect this since it renders each panel item in sequential order, however, it was bad practice, so it's all good :)